recipes/xbyak/meta.yaml

```diff
@@ -0,0 +1,44 @@
+{% set name = "xbyak" %}
+{% set version = "5.661" %}
```

Comment:
I will update the version on the feedstock, it is just I need this version for now